docs/installation.rst

```diff
@@ -114,6 +114,34 @@ Installing this package will install ``nvidia-dali-cudaXXX`` and its dependencie
   Therefore, installing the latest ``nvidia-dali-tf-plugin-cudaXXX``, will replace any older ``nvidia-dali-cudaXXX`` version already installed.
   To work with older versions of DALI, provide the version explicitly to the ``pip install`` command.
 
+nvidia-dali-video
+^^^^^^^^^^^^^^^^^
+
+DALI now ships its new video decoding functionality as a separate DALI plugin distributed as a source distribution.
+To install it, first install NVIDIA DALI, then:
+
+.. code-block:: bash
+
+   pip install scikit-build cmake ninja  # dependencies to be able to build the plugin
```

Comment:
Mhhh, that is not great. Why the build process cannot pull it automatically as a build dependency?
Maybe you can check https://packaging.python.org/en/latest/guides/writing-pyproject-toml/ to see how the build env can be defined (and the dependencies are installed in a separate env just for the build and they don't pollute the user env).

Reply:
It can, but then it can not rely on the DALI installed in the system (and we need it) to build the plugin. If I choose to make a separate build environment, then I need to put nvidia-dali as a build dependency as well, and then it comes the questions, should it be cuda12 or cuda11? Do you have a good idea how to solve this issue?

Reply:
> It can, but then it can not rely on the DALI installed in the system (and we need it) to build the plugin
Good point. After all the solution you proposed may be the best from options on the table after all.
> should it be cuda12 or cuda11? Do you have a good idea how to solve this issue?
I'm afraid I don't.